libs/datasets/statistical_areas.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
+from typing import Mapping
+import pandas as pd
+
+from libs import pipeline
+from libs.datasets import timeseries
+from libs.datasets.timeseries import MultiRegionTimeseriesDataset
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets import dataset_utils
+
+
+CBSA_LIST_PATH = "data/census-msa/list1_2020.xls"
+
+
+CBSA_COLUMN = "CBSA"
+
+
+@dataclass
+class CountyAggregator:
+    # Map from 5 digit county FIPS to 5 digit CBSA Code
+    county_map: Mapping[str, str]
+
+    # Map from 5 digit CBSA code to CBSA title
+    cbsa_title_map: Mapping[str, str]
+
+    def aggregate(self, dataset_in: MultiRegionTimeseriesDataset) -> MultiRegionTimeseriesDataset:
+        # Make a copy to avoid modifying the input. DataFrame.assign is an alternative but the API
+        # doesn't work well here.
+        df = dataset_in.data.copy()
+        df[CBSA_COLUMN] = df[CommonFields.FIPS].map(self.county_map)
+
+        # TODO(tom): Put the title in the data.
+        df_cbsa = df.groupby([CBSA_COLUMN, CommonFields.DATE], as_index=False).sum()
```

Comment:
Are you planning on adding the logic for metric aggregation here? i.e. we may not want to sum current hospitalizations if one area in the group doesn't have them

Reply:
Yes, extended the TODO. For now just trying to get all the parts wired up.